src/dotnet/Chat.Service/ChatAuthors.cs

```diff
@@ -140,6 +150,23 @@ public virtual async Task AddToContacts(IChatAuthors.AddToContactsCommand comman
         _ = await UserContactsBackend.GetOrCreate(user2Id, userId1, cancellationToken).ConfigureAwait(false);
     }
 
+    public virtual async Task CreateChatAuthors(IChatAuthors.CreateChatAuthorsCommand command, CancellationToken cancellationToken)
+    {
+        if (Computed.IsInvalidating())
+            return; // It just spawns other commands, so nothing to do here
+
+        var chatUserIds = await Backend.GetUserIds(command.ChatId, cancellationToken).ConfigureAwait(false);
+        var existingUserIds = new HashSet<string>(chatUserIds, StringComparer.Ordinal);
+        foreach (var userId in command.UserIds) {
+            if (existingUserIds.Contains(userId)) {
+                continue;
```

Comment:
We don't add brackets for single-statement block.

Reply:
done